lib/Ohmbrewer_Thermostat.cpp

```diff
@@ -227,9 +340,10 @@ void Ohmbrewer::Thermostat::parseArgs(const String &argsStr, Ohmbrewer::Equipmen
  */
 const int Ohmbrewer::Thermostat::setState(const bool state) {
     unsigned long start = millis();
-
+//    setState(state);
```

Comment:
Either do what I put for this in RIMS, or, given that getState always returns the status of the subsystems, we can ignore the state variable for both and completely remove it from setState

If you want to do the latter, it would be good to initialize _state to NULL so any fool who tries to query it directly has a bad time.